ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/layout/TutorialLiveShareLayout.kt

```diff
@@ -0,0 +1,153 @@
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import com.airbnb.lottie.compose.LottieAnimation
+import com.airbnb.lottie.compose.LottieCompositionSpec
+import com.airbnb.lottie.compose.LottieConstants
+import com.airbnb.lottie.compose.animateLottieCompositionAsState
+import com.airbnb.lottie.compose.rememberLottieComposition
+import org.ccci.gto.android.common.androidx.compose.foundation.text.minLinesHeight
+import org.cru.godtools.base.ui.theme.GodToolsTheme
+import org.cru.godtools.tutorial.Page
+import org.cru.godtools.tutorial.R
+
+@Composable
+internal fun TutorialLiveShareLayout(
+    nextPage: () -> Unit = {},
+    onTutorialAction: (Int) -> Unit = {},
+    page: Page,
+    anim: Int,
+    title: Int,
+    body: Int,
+    body2: Int? = null
+) = GodToolsTheme() {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(
+                top = dimensionResource(R.dimen.tutorial_page_inset_top),
+                bottom = dimensionResource(R.dimen.tutorial_page_inset_bottom)
+            ).verticalScroll(rememberScrollState())
+    ) {
+        Spacer(modifier = Modifier.weight(1f))
+        Box(modifier = Modifier.fillMaxWidth()) {
+            Column() {
+                Spacer(modifier = Modifier.height(290.dp))
+                Spacer(modifier = Modifier.minLinesHeight(1, MaterialTheme.typography.titleLarge))
+                Spacer(modifier = Modifier.minLinesHeight(8, MaterialTheme.typography.bodyMedium))
+            }
+
+            Column(
+                horizontalAlignment = Alignment.CenterHorizontally,
+                modifier = Modifier.fillMaxWidth()
+            ) {
+
+                Text(
+                    text = stringResource(title),
+                    style = MaterialTheme.typography.titleLarge,
+                    textAlign = TextAlign.Center,
+                    modifier = Modifier.padding(horizontal = 32.dp)
+                )
+
+                Text(
+                    stringResource(body),
+                    style = MaterialTheme.typography.bodyMedium,
+                    textAlign = TextAlign.Center,
+                    modifier = Modifier.padding(horizontal = 32.dp)
+                )
+
+                if (body2 != null) {
+                    Spacer(
+                        modifier = Modifier.minLinesHeight(
+                            minLines = 1,
+                            textStyle = MaterialTheme.typography.bodyMedium
+                        )
+                    )
+                    Text(
+                        stringResource(body2),
+                        style = MaterialTheme.typography.bodyMedium,
+                        textAlign = TextAlign.Center,
+                        modifier = Modifier.padding(horizontal = 32.dp)
+                    )
+                }
```

Comment:
I don't think there is ever a second paragraph of text in the live share tutorial